.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
     extends: 'expensify',
     parser: 'babel-eslint',
-    ignorePatterns: ['src/vendor', '!.github/actions/*.js'],
+    ignorePatterns: ['src/vendor', '!.github/actions/**/*.js'],
```

Comment:
This still isn't linting the js files in `.github/actions/` 😓

Reply:
Sounds like a blocker for the external QA actions, right?

Reply:
Potentially. Probably not a blocker per-se, but definitely a nice-to-have.

Reply:
```suggestion
    ignorePatterns: ['!.*', 'src/vendor', '.github/actions/**/index.js'],
```